assets/js/modules/adsense/util/index.js

```diff
@@ -20,6 +20,7 @@
  * External dependencies
  */
 import each from 'lodash/each';
+import { stringToDate } from '../../../util/date-range/string-to-date';
```

Comment:
@sashadoes Please add a comment for "Internal dependencies" similar to what we have in other files.